assets/main.js

```diff
@@ -51,22 +51,22 @@ function setBackground () {
 }
 //set Background
 setBackground();
-$("#backgroundBtn").on("click", function() {
+$("#backgroundBtn").on("click", function () {
     setBackground();
 })
 
-function displayQuote () {
+function displayQuote() {
 
-var quoteURL = "https://favqs.com/api/qotd";
+    var quoteURL = "https://favqs.com/api/qotd";
```

Comment:
if you format the entire page ..on a mac it is.. shift alt f ...idk about a windows... but do it before you save it because it aligns everything properly

Reply:
maybe it is formatted... and that's why it shifted .. what do you think?